<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtreeAppClass</class>
 <widget class="QMainWindow" name="RtreeAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RtreeApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>actions</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
    <addaction name="actionFind"/>
    <addaction name="actionGenerate"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>remove</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>edit</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>find</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>generate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rtreeapp.qrc"/>
 </resources>
 <connections/>
</ui>
